usr/sbin/pkcstok_migrate/pkcstok_migrate.c

```diff
@@ -1251,8 +1259,19 @@ static CK_RV identify_token(CK_SLOT_ID slot_id, char *conf_dir,
         goto done;
     }
 
-    strncpy(dll_name, stdll, max_cpy_size);
-    dll_name[max_cpy_size - 1] = 0;
+    c = confignode_find(slot->value, "usergroup");
+    if (c != NULL && (usergroup = confignode_getstr(c)) == NULL) {
+        TRACE_ERROR("failed to find usergroup for slot %lu in config file %s\n",
```

Comment:
Failed (with a capital F) at the beginning of the trace msg?

Reply:
Yep, fixed all similar error cases above as well.